lib/dogapi/facade.rb

```diff
@@ -682,6 +687,108 @@ def delete_integration(source_type_name)
       @integration_svc.delete_integration(source_type_name)
     end
 
+    #
+    # AWS INTEGRATION
+    #
+    def aws_list
```

Comment:
why not `aws_integration_list`?

Reply:
@gzussa I'm happy to go through and make all these naming convention changes. However, could you please explain to me the reasoning behind this so I can better understand for the future? Thanks!

Reply:
The reasoning is just consistency. Names should be the same (as much as possible) with what we have in the doc and other sdks.

Reply:
Gotcha, thanks for clarifying